lib/tasks/project_summary.rake

```diff
@@ -0,0 +1,11 @@
+namespace :project_summary do
+
+  desc "send daily summary of projects to registered users"
+  task daily_project_summary: :environment do
+       @users = User.all
```

Comment:
Here the user are all or DO u want me to filter users who havent registered or havent logged in once.